test/testPermissions.js

```diff
@@ -222,19 +222,19 @@ describe('permissions', function() {
 
   describe('errorResponse', function() {
     it('returns false if no error', function() {
```

Comment:
```suggestion
    it('calls next callback with true if no error', function() {
```
`it` statement doesn't match the test behaviour.